orchestra/static/orchestra/todos/todo-list.html

```diff
@@ -15,35 +15,95 @@ <h3>
+                  default-hour="8"
+                  ></time-input>
+              </div>
+              <div class="new-todo__description__datetime">
+                <label>Due</label>
+                <date-picker
+                  date="todoList.newTodoDueDate"
+                  callback="todoList.setTimeOfDate"></date-picker>
+                <time-input
+                  datetime="todoList.newTodoDueDate"
+                  default-hour="18"
+                  ></time-input>
+              </div>
+            </div>
+          </div>
+          <div class="new-todo__box">
+            <button type="submit"
+               class="btn btn-primary btn-sm edit-save-handle"
+               ng-disabled="!todoList.canAddTodo()"
+               ng-click="todoList.addTodo()">
+              Add
+            </button>
+          </div>
         </form>
       </div>
 
       <div class="col-lg-12 col-md-12 col-sm-12">
-        <div class="existing-todos">
-          <div class="todo"
-               ng-repeat="todo in todoList.todos">
-            <label>
-              <input type="checkbox"
-                     ng-model="todo.completed"
-                     ng-click="todoList.updateTodo(todo)">
-              <span class="todo__role">{{todoList.steps[todoList.taskSlugs[todo.task]].name}}:&nbsp;</span>
-              <span class="todo__description">{{todo.description}}</span>
-            </label>
-          </div>
+          <div class="existing-todos">
+            <div class="list-by-status" ng-if="(todoList.todos | filter:{completed: false}).length > 0">
+              <h4>Incomplete</h4>
+              <div class="todo"
+                   ng-repeat="todo in todoList.todos | filter:{completed: false}">
+                <label>
+                  <input type="checkbox"
+                         ng-model="todo.completed"
+                         ng-click="todoList.updateTodo(todo)">
+                  <span class="todo__role">{{todoList.steps[todoList.taskSlugs[todo.task]].name}}:&nbsp;</span>
+                  <span class="todo__description">{{todo.description}}</span>
+                  <span class="todo__dates">
+                    <span ng-if="todoList.isNonEmptyString(todo.start_by_datetime)">
+                      Start {{todoList.getDateDisplay(todo.start_by_datetime)}}
+                    </span>
+                    <span ng-if="todoList.isNonEmptyString(todo.start_by_datetime) && todoList.isNonEmptyString(todo.due_datetime)" class="todo__dates-separator">|</span>
+                    <span ng-if="todoList.isNonEmptyString(todo.due_datetime)">
+                      Due {{todoList.getDateDisplay(todo.due_datetime)}}</span>
+                  </span>
+                </label>
+              </div>
+            </div>
+            <div class="list-by-status" ng-if="(todoList.todos | filter:{completed: true}).length > 0">
+              <h4>Complete</h4>
+              <div class="todo"
```

Comment:
the inners of these are repetitive.

make a directive, or import a shared html file?